bgpd/bgp_route.c

```diff
@@ -9364,6 +9364,15 @@ static void route_vty_short_status_out(struct vty *vty,
 	enum rpki_states rpki_state = RPKI_NOT_BEING_USED;
 
 	if (json_path) {
+		/* RPKI validation state */
+		rpki_state = hook_call(bgp_rpki_prefix_status, path->peer, path->attr, p);
```

Comment:
Can we write this (at the top of the function) once (instead of duplicating what is already for non-JSON output)?

Reply:
done